src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -185,7 +185,7 @@ function AddEditKeyHandlers(container) {
         e.preventDefault();
         document.execCommand("justifycenter", false, null);
     });
-    
+
     //Note, CTRL+N is also caught, but up on the Shell where it is turned into an event, so that it can be caught even when the focus isn't on the browser
```

Comment:
An inadvertent change like this can always be reverted by "get checkout -- <filepath>" before commiting.  :-)
Of course, if you really wanted to get rid of that whitespace...